### PR TITLE
Fix RPA validation errors from konflux-release-data validations

### PR DESCRIPTION
**Pull request by creydr**
* Add `labels`
* Move `defaults` under `.data.mapping`
* Add release notes section
* Use correct `sign.configMapName` in FBC RPA
* Add missing `---` at first line
* Remove empty blank lines
* Use `fbc-stage` policy for stage FBC RPAs

### CODE REVIEW

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

/hold if you want to wait for the downstream sync to be merged by releng team

File: pkg/konfluxgen/konfluxgen.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"embed"
 	"encoding/json"
 	"fmt"
+	"github.com/blang/semver/v4"
```
[the remark this answers is not in the document]
Comment (creydr) [reply pkg/konfluxgen/konfluxgen.go]: Updated

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**